src/components/RichTextPopup/RichTextPopup.js

```diff
@@ -135,6 +141,17 @@ const RichTextPopup = () => {
   const syncDraggablePosition = (e, { x, y }) => {
     setDraggablePosition({ x, y });
   };
+
+
+  const insertSymbols = symbol => {
+    const contents = editorRef.current.getContents();
+    const { index, length } = editorRef.current.editor.getSelection();

```

Comment:
`current.editor.getSelection` will be gone in the production build. I think we should use `current.getSelection` instead.